app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -131,6 +131,79 @@ def test_bugzilla_error_creates_error_notification(self):
         )
 
 
+class TestCommentDSBugTask(MockRequestMixin, MockBugzillaMixin, TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        self.experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_SHIP
+        )
+        self.experiment.save()
+
+    def test_ds_bugs_successfully_updates_with_comment(self):
+        self.assertEqual(Notification.objects.count(), 0)
+
+        tasks.update_ds_bug_task(self.experiment.id)
+
+        self.mock_bugzilla_requests_post.assert_called()
```

Comment:
Do you wanna check the parameters here using `assert_called_with`?  Looks like we do that some places and not others?  We don't have to, just curious.

Reply:
@jaredkerim would probably be best `assert_called()` were replaced with `assert_called_with` created https://github.com/mozilla/experimenter/issues/1628 to cover that as `assert_called()` has been used outside of this PR